python/spear/config/default_config.simulation_controller.yaml

```diff
@@ -15,6 +15,14 @@ SIMULATION_CONTROLLER:
   PORT: 30000
 
   SIMULATION_STEP_TIME_SECONDS: 0.1
+  ENABLE_ENHANCED_DETERMINISM: True
+  ENABLE_SUBSTEPPING: True
+  MAX_SUBSTEP_DELTA_TIME: 0.01
+  MAX_SUBSTEPS: 100
+  ContactOffsetMultiplier: 0.010000
```

Comment:
CONTACT_OFFSET_MULTIPLIER (we should stick to our all-caps naming convention)

Reply:
move this section of parameters to be after CUSTOM_UNREAL_CONSOLE_COMMANDS but before CAMERA_AGENT

Reply:
no need for enhanced determinism and substepping because updating them here has no effect

Reply:
no need to say 0.0100000, just say 0.01

Reply:
in default_config, set {ContactOffsetMultiplier, MinContactOffset, MaxContactOffset} to their default values in the Unreal Engine, and then set them to these weird values per-example on an as-needed basis (do we need these values for the OpenBot example?)

Reply:
> no need for ENABLE_ENHANCED_DETERMINISM and ENABLE_SUBSTEPPING because we found that updating them in the yaml has no effect

They do have an effect when updated via yaml file. It's just that the values need to be set initially in config file for the current set of actions in the urdf open loop control example to work.

Reply:
> in default_config, set {ContactOffsetMultiplier, MinContactOffset, MaxContactOffset} to their default values in the Unreal Engine, and then set them to these weird values per-example on an as-needed basis (do we need the weird values for the OpenBot example?)

I'm not sure if we need these for OpenBot example. Not setting these weird values did not seem to affect the way OpenBot navigated in the environment.